azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/DeployMojo.java

```diff
@@ -44,29 +45,27 @@
     public static final String FUNCTION_APP_UPDATE = "Updating Azure Function App...";
     public static final String FUNCTION_APP_UPDATE_DONE = "Successfully updated Azure Function App ";
 
-    public static final String MS_DEPLOY = "msdeploy";
-    public static final String FTP = "ftp";
-
     //region Properties
 
     /**
      * Deployment type to deploy Web App. Supported values:
      * <ul>
      * <li>msdeploy</li>
      * <li>ftp</li>
+     * <li>zip</li>
      * </ul>
      *
      * @since 0.1.0
      */
-    @Parameter(property = "functions.deploymentType", defaultValue = MS_DEPLOY)
+    @Parameter(property = "functions.deploymentType", defaultValue = "msdeploy")
```

Comment:
use zip as default

Reply:
The common practice is to use enum type internally. Using string is a downgrade.
There should be some auto-converting mechanism (serialization/deserialization) to automatically convert from string to enum type. The result is that we use enum value directly in the function body.

This is the next level of excellence we want to push.

Reply:
@jdneo  I will make that change in next PR.

@akaroml  I tried, by overriding the toString function of an enum value. But problem here is that, the default value requires a constant. That is the one I didn't figure out. I will keep trying.